faker.go

```diff
@@ -285,3 +290,9 @@ func NewWithSeed(src rand.Source) (f Faker) {
 	f = Faker{Generator: generator}
 	return
 }
+
+func newWithGenerator(generator RandomGenerator) (f Faker) {
```

Comment:
considering that we already have the `faker.NewWithSeed()` to have know seeds (akka expected generated values), I guess we don't need this part.